src/misc_early.rs

```diff
@@ -113,4 +125,13 @@ impl EarlyLintPass for MiscEarly {
             }
         }
     }
+
+    fn check_expr(&mut self, cx: &EarlyContext, expr: &Expr) {
+        if let ExprKind::Call(ref path, ref args) = expr.node {
+            if let ExprKind::Closure(_, _, ref block) = path.node {
```

Comment:
I’m sorry if I mislead you with my previous comment (I edited it a few minutes after I actually checked the facts :sweat_smile:) but you do need a `ExprKind::Paren` here between those two. (We never get `Paren` in `LateLintPass`). Otherwise it works.
Also the name `path` here is not a good name as in that specific case the function is not a path but a closure.